src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -274,3 +305,5 @@ const getQueryVariables = (
     execution: queryParamAsNumber(execution),
   };
 };
+
+const isLobsterLink = (url: string) => url.split("/")[3] === "lobster";
```

Comment:
assumes that all urls with a pathname that starts with /lobster and have the protocol included are links to lobster

Reply:
Can we write tests for this function this will break if there isn't a 4th `/` in the url